docusaurus/docs/operate/Getting Started/1_quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 1
+title: Quick Start Guide (<10 minutes)
+description: Guide to get a PATH instance up and running.
+---
+
+This guide will help you set up and run PATH to serve requests using Morse or Shannon protocol in under 10 minutes.
+
+> **Note:** This guide covers running PATH without any authentication or authorization mechanisms. Work is underway to provide these capabilities - see [PATH Guard documentation](https://path.grove.city/operate/helm/guard) for details.
+
+## Prerequisites
+
+- Docker installed and running on your system
+
+## 1. Prepare Your Configuration
+
+First, prepare your configuration file by following [the PATH Config File instructions](https://path.grove.city/develop/path/configurations_path).
+
+1. Create a config directory for PATH:
+   ```bash
+   mkdir -p ./path/config
+   ```
+
+2. Copy your configuration file to the new directory with the correct name:
+   ```bash
+   export CONFIG_FILE=/PATH/TO/YOUR/CONFIG/FILE
+   cp $CONFIG_FILE ./path/config/.config.yaml
+   ```
+
+## 2. Set Up the PATH Container
+
+1. Set the PATH container image version from [the set of available tags](https://github.com/buildwithgrove/path/pkgs/container/path):
+   ```bash
+   export PATH_IMAGE_TAG='sha-9a877a0-rc'
```

Comment:
Can we include reference to using the `main` tag and a TODO to update it to `latest` once the release artifacts CI is finished?